packages/web-components/src/components/masthead/masthead-contact.ts

```diff
@@ -45,6 +47,18 @@ class DDSMastheadContact extends DDSMastheadProfile {
     `;
   }
 
+  updated(changedProperties) {
+    if (changedProperties.has('expanded')) {
+      if (!this.expanded) {
+        const mastheadContainer = this.closest(`${ddsPrefix}-masthead-container`) as DDSMastheadContainer;
+
+        if (mastheadContainer.contactModuleApp) {
+          (mastheadContainer.contactModuleApp as CMApp).init();
```

Comment:
Maybe worth a comment explaining the workaround and that the team that owns the content module is working on a better API to do this? It's confusing reading a call to `init()` when what we mean here is more like minimize.

Reply:
Good call.